en/backup-restore-overview.md

```diff
@@ -27,268 +33,51 @@ TiDB Operator 1.1 and later versions implement the backup and restore methods us
-    - When using Lightning for restoring, you can specify the Lightning version in this field. For example, `spec.toolImage: pingcap/lightning:v5.4.0`. If not specified, the Lightning version specified in `TOOLKIT_VERSION` of the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) is used for restoring by default.
-    - TiDB Operator supports this configuration starting from v1.1.9.
-* `.spec.to.host`: The address of the TiDB cluster to be restored.
-* `.spec.to.port`: The port of the TiDB cluster to be restored.
-* `.spec.to.user`: The accessing user of the TiDB cluster to be restored.
-* `.spec.to.secretName`: The secret that contains the password of the `.spec.to.user`.
-* `.spec.to.tlsClientSecretName`: The secret of the certificate used during the restore.
+- [Back up Data to S3-Compatible Storage Using BR](backup-to-aws-s3-using-br.md)
+- [Back up Data to GCS Using BR](backup-to-gcs-using-br.md)
+- [Back up Data to PV Using BR](backup-to-pv-using-br.md)
 
-    If [TLS](enable-tls-between-components.md) is enabled for the TiDB cluster, but you do not want to restore data using the `${cluster_name}-cluster-client-secret` as described in [Enable TLS between TiDB Components](enable-tls-between-components.md), you can use the `.spec.to.tlsClient.tlsSecret` parameter to specify a secret for the restore. To generate the secret, run the following command:
+If you have the following backup needs, you can use Dumpling to make a backup of the TiDB cluster data:
 
-    {{< copyable "shell-regular" >}}
+- To export SQL or CSV files
+- To limit the memory usage of a single SQL statement
+- To export the historical data snapshot of TiDB
 
-    ```shell
-    kubectl create secret generic ${secret_name} --namespace=${namespace} --from-file=tls.crt=${cert_path} --from-file=tls.key=${key_path} --from-file=ca.crt=${ca_path}
-    ```
+Refer to the following documents for more information:
 
-* `.spec.storageClassName`: The persistent volume (PV) type specified for the restore operation.
-* `.spec.storageSize`: The PV size specified for the restore operation. This value must be greater than the size of the backup data.
-* `.spec.tableFilter`: Specifies tables that match the [table filter rules](https://docs.pingcap.com/tidb/stable/table-filter/) for BR. This field can be ignored by default.
+- [Back up Data to S3-Compatible Storage Using Dumpling](backup-to-s3.md)
+- [Back up Data to GCS Using Dumpling](backup-to-gcs.md)
 
-    If the field is not configured, the default `tableFilter` value for TiDB Lightning is as follows:
+### Restore data
 
-    ```bash
-    tableFilter:
-    - "*.*"
-    - "!/^(mysql|test|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA)$/.*"
-    ```
+If you need to recover the SST files exported by BR to a TiDB cluster, you can use BR. Refer to the following documents for more information:
```

Comment:
for `you *can* use BR`, is `must`/`should` better?

Reply:
It depends on whether this action is required or optional. If it's required, use *must*; otherwise, use *can*. Is this a **required** action?

Note that *should* is generally avoided. See [should](https://developers.google.com/style/word-list#should).

Reply:
**only** BR can be used to `recover the SST files exported by BR to a TiDB cluster`

Reply:
Got it. Thanks.